data_override/data_override.F90

```diff
@@ -692,610 +230,53 @@ subroutine data_override_3d(gridname,fieldname_code,data,time,override,data_inde
   logical,           optional, intent(out) :: override !< true if the field has been overriden succesfully
   type(time_type),              intent(in) :: time !< (target) model time
   integer,           optional,  intent(in) :: data_index
-  real, dimension(:,:,:),    intent(inout) :: data !< data returned by this call
+  class(*), dimension(:,:,:),    intent(inout) :: data !< data returned by this call
```

Comment:
class(*) should be a macro?

Reply:
This subroutine was removed in b89c8db621164eef22e0d9220a2046a4a1ceaa98.